spec/CloudCode.spec.js

```diff
@@ -3168,3 +3168,29 @@ describe('afterLogin hook', () => {
     await query.find({ context: { a: 'a' } });
   });
 });
+
+describe('sendMail', () => {
```

Comment:
See [this comment](https://github.com/parse-community/docs/pull/806#discussion_r548999827); please change to `send email`.